docs/developer-guide/nodejs-api.md

```diff
@@ -1382,17 +1382,17 @@ ruleTester.run("my-rule", myRule, {
 [builtin-formatters]: https://eslint.org/docs/user-guide/formatters/
 [thirdparty-formatters]: https://www.npmjs.com/search?q=eslintformatter
 [eslint]: #eslint-class
-[eslint-constructor]: #-new-eslintoptions
-[eslint-lintfiles]: #-eslintlintFilespatterns
-[eslint-linttext]: #-eslintlintTextcode-options
-[eslint-calculateconfigforfile]: #-eslintcalculateConfigForFilefilePath
-[eslint-ispathignored]: #-eslintisPathIgnoredfilePath
-[eslint-loadformatter]: #-eslintloadFormatternameOrPath
-[eslint-version]: #-eslintversion
-[eslint-outputfixes]: #-eslintoutputFixesresults
-[eslint-geterrorresults]: #-eslintgetErrorResultsresults
-[lintresult]: #-lintresult-type
-[lintmessage]: #-lintmessage-type
-[editinfo]: #-editinfo-type
-[formatter]: #-formatter-type
+[eslint-constructor]: #new-eslint-options
+[eslint-lintfiles]: #eslint-lintfiles-patterns
+[eslint-linttext]: #eslint-linttext-code-options
+[eslint-calculateconfigforfile]: #eslint-calculateconfigforfile-filepath
+[eslint-ispathignored]: #eslint-ispathignored-filepath
+[eslint-loadformatter]: #eslint-loadformatter-nameorpath
+[eslint-version]: #eslint-version
+[eslint-outputfixes]: #eslint-outputfixes-results
+[eslint-geterrorresults]: #eslint-geterrorresults-results
```

Comment:
These should be reverted to work well on GitHub (we should just convert uppercase to lowercase).